libs/components/lookup/src/lib/modules/lookup/lookup.component.spec.ts

```diff
@@ -562,6 +562,7 @@ describe('Lookup component', function () {
           fixture.detectChanges();
           tick();
           fixture.detectChanges();
+          tick();
```

Comment:
All the changes here and in the autocomplete spec were to fix timing issues that were caused by moving the timing of the event that repositions things.